market_communication/boney_comb_transaktionsdaten.go

```diff
@@ -32,6 +32,20 @@ const dokumentnummerKey = "Dokumentennummer"
 // nachrichtendatumKey is the key under which the Nachrichtendatum is stored in the BOneyComb.Transaktionsdaten
 const nachrichtendatumKey = "Nachrichtendatum"
 
+// nachrichtendatumKey is the key under which the Nachrichtendatum is stored in the BOneyComb.Transaktionsdaten
+const pruefidentifikatorKey = "Pruefidentifikator"
+
+// GetPruefidentifikator returns the Pruefidentifikator from BOneyComb.Transaktionsdaten if it's present and nil otherwise
+func (boneyComb *BOneyComb) GetPruefidentifikator() *string {
+	return boneyComb.GetTransactionData(pruefidentifikatorKey)
+}
+
+// SetPruefidentifikator sets the Pruefidentifikator in the BOneyComb.Transaktionsdaten
+func (boneyComb *BOneyComb) SetPruefidentifikator(pruefi string) {
+	boneyComb.initializeTransaktionsdaten()
+	boneyComb.Transaktionsdaten[pruefidentifikatorKey] = pruefi
+}
```

Comment:
Is this getter and setter pattern also a go pattern?

Reply:
None that I'm aware of ;) 
It's basically the workaround/helper method for writing/reading data from the untyped Transaktionsdaten without worrying about the exact key ("pruefi" vs. "Prüfi" vs. "Pruefidentifikator" vs. "prüfidentifiaktor").
The intention is to make access to common transaktionsdaten easy and failsafe by hiding the dictionary-aspect and casesensistivy of the keys behind well defined methods.